Dalamud/Interface/Internal/DesignSystem/DalamudComponents.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Dalamud.Interface.Internal.DesignSystem;
+
+/// <summary>
+/// Private ImGui widgets for use inside Dalamud.
+/// </summary>
+internal static partial class DalamudComponents
+{
+}
```

Comment:
Intended empty file?

Reply:
Yeah, we do it the same way for the other components class.